data/World/Fire Temple MQ.json

```diff
@@ -45,15 +67,30 @@
                 has_fire_source and has_explosives and
                 (can_use(Hookshot) or logic_fire_mq_blocked_chest)",
             "Fire Temple MQ GS Big Lava Room Open Door": "True",
+            "Fire Temple MQ Big Lava Room Left Pot": "True",
+            "Fire Temple MQ Big Lava Room Right Pot": "(can_use(Hookshot) or logic_fire_mq_blocked_chest)",
+            "Fire Temple MQ Big Lava Room Alcove Pot": "True",
+            "Fire Temple MQ Boss Key Chest Room Pot": "
+                has_fire_source and (Bow or logic_fire_mq_bk_chest) and can_use(Hookshot)",
             "Fairy Pot": "
                 has_bottle and has_fire_source and (Bow or logic_fire_mq_bk_chest) and
                 (can_use(Hookshot) or logic_fire_song_of_time)"
 
         },
         "exits": {
-            "Fire Lower Maze": "
-                can_use(Goron_Tunic) and (Small_Key_Fire_Temple, 2) and
-                (has_fire_source or (logic_fire_mq_climb and Hover_Boots))"
+            "Fire Elevator Room": "can_use(Goron_Tunic) and (Small_Key_Fire_Temple, 2)"
+        }
+    },
+    {
+        "region_name": "Fire Elevator Room",
+        "dungeon": "Fire Temple",
+        "locations": {
+            "Fire Temple MQ Elevator Room Recovery Heart 1" : "True",
+            "Fire Temple MQ Elevator Room Recovery Heart 2" : "True",
+            "Fire Temple MQ Elevator Room Recovery Heart 3" : "True"
```

Comment:
Again just like in vanilla there is a possibility of the recovery hearts in the fire elevator room being safe under FTR, particularly if you have the Longshot. I think I might ask the person who did the ill-fated DMC rework to weigh in?